setup_proxmox.sh

```
@@ -0,0 +1,630 @@
+    log "Node.js kuruldu: $NODE_VERSION"
+    log "npm kuruldu: $NPM_VERSION"
+    
+    # Check minimum versions
+    NODE_MAJOR=$(echo $NODE_VERSION | cut -d. -f1 | tr -d 'v')
+    if [ "$NODE_MAJOR" -lt 20 ]; then
+        error "Node.js sürümü çok eski ($NODE_VERSION). En az v20.0.0 gerekli."
+    fi
+}
+
+# Install PM2 for process management
+install_pm2() {
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)